webpages/styles/colors.css

```diff
@@ -18,19 +18,21 @@
   --dark-red-text: var(--dark-red-variant);
   --white-text: #fff;
 
-  --page-background: #222;
-  --content-background: #2a2a2a;
+  --page-background: #181818;
+  --content-background: #272727;
   --navigation-background: #333;
-  --search-toolbar-background: #2a2a2a;
-  --button-background: #222;
+  --search-toolbar-background: #1c1c1c;
+  --button-background: #181818;
   --button-hover-background: #1a1a1a;
-  --input-background: #222;
-  --search-input-background: #111;
+  --tab-hover-underline: #1e1e1e;
+  --input-background: #1d1d1d;
+  --search-input-background: #272727;
   --switch-border: #808080;
   --switch-inner-background: #eee;
   --tooltip-background: #000;
-  --comment-background: #222;
-  --unread-comment-background: #000;
+  --comment-background: #1c1c1c;
+  --unread-comment-background: #0d0d0d;
+  --unread-comment-border: inset 0px 0px 0px 2px #161616;
   --notice-background: var(--navigation-background);
```

Comment:
Could this be made yellow? It's hard to tell the difference between normal and yellow notices right now.

Reply:
wdym by "yellow notices" ?

Reply:
Stable: 
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/87dd331e-f355-43b1-aa7b-964b6dce799d)
PR:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/81489795/4402053d-71cc-4897-97ba-4bd295da170a)

Reply:
> Stable: ![image](https://private-user-images.githubusercontent.com/81489795/311513770-87dd331e-f355-43b1-aa7b-964b6dce799d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzgyNTYsIm5iZiI6MTcxMDA3Nzk1NiwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTM3NzAtODdkZDMzMWUtZjM1NS00M2IxLWFhN2ItOTY0YjZkY2U3OTlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzMzkxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViYjI1MTA3NmM3ZTE3MjJiZTAxMmE1NmViNDZlMThlNTI1ZDFmNjZkODYyZDc0ZjY2MzA1OWU5ZGI1ZmYxZmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zWCVWqM9BBYQ-mazo_lI1dy4sVdyDECtoUNx2syxDTc) PR: ![image](https://private-user-images.githubusercontent.com/81489795/311513785-4402053d-71cc-4897-97ba-4bd295da170a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAwNzgyNTYsIm5iZiI6MTcxMDA3Nzk1NiwicGF0aCI6Ii84MTQ4OTc5NS8zMTE1MTM3ODUtNDQwMjA1M2QtNzFjYy00ODk3LTk3YmEtNGJkMjk1ZGExNzBhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEwVDEzMzkxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwZmEwNGY2NzkxODJlMjE4OTdlMDA3YTRmOGEyODM4NGRhZTgyODhhY2VjZWYyNjg3NmFmMDYzMjc4ZTEzNjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Y148jv-GlVhlzZM1JInCSPcXHCKAsL_F7o2VNo4rAXc)
> 
> It already has a yellow background in light mode.

Oh

Reply:
Just pushed it :)